.eslintrc.js

```diff
@@ -128,8 +129,8 @@ module.exports = {
   settings: {
     react: {
       pragma: 'React',
-      version: '15.0',
-      flowVersion: '0.63.1',
+      version: '17.0',
+      flowVersion: '0.96.0',
```

Comment:
I'm not exactly sure about the usefulness of these properties, but seeing them outdated always makes me crazy so I took the opportunity to update these. It's supposed to trigger different behaviors in the eslint react plugin, but I'm not so sure.

Reply:
I'm not sure if it's because of this, but I started to get 4 additional warnings when I tested this on my local.
```
profiler/src/test/components/StackChart.test.js
  240:3  warning  Test has no assertions  jest/expect-expect
  244:3  warning  Test has no assertions  jest/expect-expect

profiler/src/test/components/TrackContextMenu.test.js
  428:5  warning  Test has no assertions  jest/expect-expect
  540:5  warning  Test has no assertions  jest/expect-expect
```

Edit: Well, actually it looks like I'm getting that on the main branch as well. I guess I didn't see them earlier because I didn't do a `yarn install` properly earlier 🤷‍♂️ . I guess we can fix this in another PR.

Reply:
Yeah, I filed this yesterday about that: https://github.com/jest-community/eslint-plugin-jest/issues/988
I think this is due to the latest upate...
This is unfortunate because they skip it for "todo", see https://github.com/jest-community/eslint-plugin-jest/pull/954 ! Then it may be very easy to fix there...

Reply:
Oh, interesting. This is definitely something they should fix. Also, maybe we should change these tests to `it.todo` as well?

Reply:
Maybe `it.todo` would be more appropriate in this case yeah

Reply:
Yeah. Also to be clear, I wasn't talking for this PR, we can fix these in another PR.